jdaviz/cli.py

```diff
@@ -111,8 +116,8 @@ def _main(config=None):
                                      'loaded from FILENAME.')
     filepaths_nargs = '*'
     if config is None:
-        parser.add_argument('layout', choices=['cubeviz', 'specviz', 'specviz2d',
-                                               'mosviz', 'imviz'],
+        parser.add_argument('--layout', default='', choices=['cubeviz', 'specviz', 'specviz2d',
```

Comment:
This changes the CLI calls (again), so all the docs referencing CLI commands also need updating.

Reply:
Wait, does it? The existing commands should still all work. The only change here is `layout` is now an optional argument. But since it's a positional one, we don't actually have docs that use `--layout myviz`, only `jdaviz myviz`, which should still work

Reply:
But someone doing `jdaviz --help` would see a difference. And changing something from required to optional is a significant change that warrants a change log and doc updates.

Reply:
Certainly changelog, yes (I was mainly waiting to avoid merge conflicts with the changelog)! I just can't find anywhere in the docs we mentioned the config being a "required arg". The docs I see that show how to use the CLI are mainly per config docs of "use `jdaviz myviz /path/to/file.fits` to start myviz", which still works

Reply:
Another part, I don't think we have any mention in the docs for the `jdaviz.open` logic yet, which I think was intentional until we flesh out the workflow enough

Reply:
But as per this implementation, isn't `jdaviz --layout myviz /path/to/file.fits` more correct now?

Reply:
Oh did this change inadvertently drop support for `jdaviz myviz`? Let me verify

Reply:
Ok based on the team's input today, we'll change the syntax, yet again, to `jdaviz --layout=myviz` rather than add in a hack to preserve the old functionality. Done in https://github.com/spacetelescope/jdaviz/pull/2257/commits/d496d2dedecc0d75e338e9071f1d0955e00acd68